apps/site/components/ApiDocs/StabilityIndex/index.tsx

```diff
@@ -0,0 +1,19 @@
+import classNames from 'classnames';
+import type { FC, PropsWithChildren } from 'react';
+
+import styles from './index.module.css';
+
+type StabilityIndexProps = PropsWithChildren<{
+  level: number;
```

Comment:
Instead of level, can it be a string, like "label" or "title"?

Reply:
Title + level as `"stable" | "experimental"` ?

Reply:
and if we do that I can remove the change on `.stylelintrc.mjs`

Reply:
Simply make it a string. Because the label can have any string on it.

I also think we shouldn't call this StabilityIndex, let's move this component to Common and call it Alert

Reply:
Because we want to use it on the Downloads page too

Reply:
Humm so move it to comment